src/sys/socket/mod.rs

```diff
@@ -14,10 +14,9 @@ use std::net;
 #[cfg(feature = "uio")]
 use crate::sys::time::TimeSpec;
 #[cfg(feature = "uio")]
-use crate::sys::{
-    time::TimeVal,
-    uio::IoVec
-};
+use crate::sys::time::TimeVal;
+#[cfg(feature = "uio")]
```

Comment:
This line shouldn't be necessary.  It was only necessary before your change because it gated a `use crate` import.
```suggestion
```